WPMedia/fusion-news-theme-blocks
# tmedia 962 Fix update tests new sizes

## Description

Update large size in ratios for 
## Jira Ticket

- [TMEDIA-962](https://arcpublishing.atlassian.net/browse/TMEDIA-962)

## Acceptance Criteria

Top table list has bigger large image

## Test Steps

1. Checkout this branch `git checkout FIX-update-tests-new-sizes`
2. Run fusion repo with linked blocks npx fusion start -f -l @wpmedia/large-promo-block,@wpmedia/shared-styles,@wpmedia/resizer-image-block
3. Add a large promo block 
4. Add the website url of /2019/12/02/baby-panda-born-at-the-zoo/
5. See the image in all ratios available
6. Make sure there's no malformed images

## Effect Of Changes

### Before

large promo wouldn't render images from a content source 

### After

large promos will render images from a content source

## Dependencies or Side Effects

- Updated the ci to be 1.24 rather than 1.23 per the release here
- Also updated the feature pack to be 1.24 rather than 1.23 in an already merged pr

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).
- [ ] Confirmed relevant documentation has been updated/added. -> not relevant, block has placeholder here anyway

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._


- Fixes test for botched release due to ci issues https://github.com/WPMedia/arc-themes-blocks/actions/runs/3237014216/jobs/5303551315
- Fixes changeset ci issues 


todo: 

- Update all places that use the width and height of the large resizer. Any place that has the large ratio for the change. For example, if the width and height is specified in the params of the content source that previously would be the large ratio, that needs to change. Going to try to look into storybook to see if we're mocking this correctly. Storybook won't be useful here as we need live images to test with all blocks `npx fusion start -f -l`


---
